.github/workflows/release.yml

```diff
@@ -0,0 +1,52 @@
+name: Create SDK release
+
+on: 
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: macOS-latest
+    
+    steps:
+    
+    - uses: actions/checkout@master
+      with:
+        ref: 'master'
+    
+    - uses: actions/setup-ruby@v1
+      with:
+        ruby-version: '2.6' 
+        
+    - name: Install dependencies
+      run: gem install cocoapods
+    
+    - name: Create a draft release
+      uses: actions/github-script@0.9.0
```

Comment:
Just curious, could we have used the create-release action for this? https://github.com/actions/create-release#example-workflow---create-a-release

Reply:
Theoretically yes, but it's much less flexible and people have been having problems with it. Using the script lets us be a lot more explicit about what this action does and will make it obvious what can change if things need to in the future.